isthmus/src/main/java/io/substrait/isthmus/expression/FunctionMappings.java

```diff
@@ -11,111 +11,104 @@
-                resolver(
-                    SqlStdOperatorTable.MINUS_DATE, Set.of("date", "timestamp_tz", "timestamp")));
-  }
+  public static final ImmutableList<Sig> SCALAR_SIGS =
+      ImmutableList.<Sig>builder()
+          .add(
+              s(SqlStdOperatorTable.PLUS, "add"),
+              s(SqlStdOperatorTable.MINUS, "subtract"),
+              s(SqlStdOperatorTable.UNARY_MINUS, "negate"),
+              s(SqlStdOperatorTable.MULTIPLY, "multiply"),
+              s(SqlStdOperatorTable.DIVIDE, "divide"),
+              s(SqlStdOperatorTable.ABS, "abs"),
+              s(SqlStdOperatorTable.MOD, "modulus"),
+              s(SqlStdOperatorTable.POWER, "power"),
+              s(SqlStdOperatorTable.EXP, "exp"),
+              s(SqlStdOperatorTable.SIN, "sin"),
+              s(SqlStdOperatorTable.COS, "cos"),
+              s(SqlStdOperatorTable.TAN, "tan"),
+              s(SqlStdOperatorTable.ASIN, "asin"),
+              s(SqlStdOperatorTable.ACOS, "acos"),
+              s(SqlStdOperatorTable.ATAN, "atan"),
+              s(SqlStdOperatorTable.ATAN2, "atan2"),
+              s(SqlStdOperatorTable.SIGN, "sign"),
+              s(SqlStdOperatorTable.LOG10, "log10"),
+              s(SqlStdOperatorTable.LN, "ln"),
+              s(SqlStdOperatorTable.AND),
+              s(SqlStdOperatorTable.OR),
+              s(SqlStdOperatorTable.NOT),
+              s(SqlStdOperatorTable.LESS_THAN, "lt"),
+              s(SqlStdOperatorTable.LESS_THAN_OR_EQUAL, "lte"),
+              s(SqlStdOperatorTable.GREATER_THAN, "gt"),
+              s(SqlStdOperatorTable.GREATER_THAN_OR_EQUAL, "gte"),
+              s(SqlStdOperatorTable.EQUALS, "equal"),
+              s(SqlStdOperatorTable.BIT_XOR, "xor"),
+              s(SqlStdOperatorTable.IS_NULL, "is_null"),
+              s(SqlStdOperatorTable.IS_NOT_NULL, "is_not_null"),
+              s(SqlStdOperatorTable.NOT_EQUALS, "not_equal"),
+              s(SqlStdOperatorTable.MINUS_DATE, "subtract"),
+              s(SqlStdOperatorTable.DATETIME_PLUS, "add"),
+              s(SqlStdOperatorTable.EXTRACT, "extract"),
+              s(SqlStdOperatorTable.CEIL, "ceil"),
+              s(SqlStdOperatorTable.FLOOR, "floor"),
+              s(SqlStdOperatorTable.ROUND, "round"),
+              s(SqlStdOperatorTable.LIKE),
+              s(SqlStdOperatorTable.REPLACE, "replace"),
+              s(SqlStdOperatorTable.SUBSTRING, "substring"),
+              s(SqlStdOperatorTable.CONCAT, "concat"),
+              s(SqlStdOperatorTable.CHAR_LENGTH, "char_length"),
+              s(SqlStdOperatorTable.LOWER, "lower"),
+              s(SqlStdOperatorTable.UPPER, "upper"),
+              s(SqlStdOperatorTable.BETWEEN),
+              s(SqlStdOperatorTable.IS_NOT_DISTINCT_FROM, "is_not_distinct_from"),
+              s(SqlStdOperatorTable.COALESCE, "coalesce"),
+              s(SqlStdOperatorTable.TRIM, "trim"),
+              s(SqlStdOperatorTable.TRIM, "ltrim"),
+              s(SqlStdOperatorTable.TRIM, "rtrim"))
```

Comment:
These lines are the only actual change in this file. The rest is just slightly restructuring the existing code.